### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by KajalSinghBAGHEL**
remove grammatical mistakes

### CODE REVIEW

**Comment by KajalSinghBAGHEL** [comment]
I am a GSSOC participant. Please check my pull request

**Comment by KajalSinghBAGHEL** [comment]
I am a GSSOC participant. Please check my pull request

File: README.md [hunk 1]
```diff
@@ -109,3 +109,5 @@ Thanks to these wonderful people ✨✨:
 		</td>
 	</tr>
 </table>
+
+
```
Comment (czgdp1807) [inline README.md:113]:
```suggestion

```

**Comment by czgdp1807** [comment]
Thanks.

**Pull request merged**